tfx/components/transform/executor_test.py

```diff
@@ -227,51 +243,34 @@ def testCounters(self):
     self.assertMetricsCounterEqual(metrics, 'analyze_paths_count', 1)
 
   def testDoWithCache(self):
-
     # First run that creates cache.
-    output_cache_artifact = _OutputCache()
-    output_cache_artifact.uri = os.path.join(self._output_data_dir, 'CACHE')
-
-    self._output_dict['cache_output_path'] = [output_cache_artifact]
-
     self._exec_properties['module_file'] = self._module_file
     self._transform_executor.Do(self._input_dict, self._output_dict,
                                 self._exec_properties)
     self._verify_transform_outputs()
```

Comment:
Should this pass materialize_cache=True?
Same for L265 below

Reply:
by default store_cache is true in `_verify_transform_outputs()`, so no need to specify it unless we want to be more explicit.

Reply:
added the check cache tag to be explicit